packages/utilities/src/object.ts

```diff
@@ -87,7 +88,7 @@ export function filteredAssign(isAllowed: (propName: string) => boolean, target:
 export function getId(prefix?: string): string {
   let index = _global[CURRENT_ID_PROPERTY]++;
 
-  return (prefix || '') + index;
+  return (prefix || DEFAULT_ID_STRING) + index;
```

Comment:
If you don't want to change the intended output, you could rewrite it as:
```ts
return `${prefix || ''}${index}`;
```

Reply:
It's already technically returning a string. The problem was that in HTML 4 an ID was required to start with a letter. So Really I need to change the default string to be something like `id__`. 

The alternative is to not let prefix be undefined. Ideally the prefix should be a unique and descriptive identifier. However that is a breaking change.

See this [stackoverflow](https://stackoverflow.com/questions/5672903/can-i-have-a-div-with-id-as-number) for more.

![image](https://user-images.githubusercontent.com/20801176/32621762-b5471c6c-c535-11e7-935a-43c207ccb16a.png)